src/components/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2025 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import styled from '@emotion/styled';
+import React, { forwardRef } from 'react';
+import { IconSlash } from '../icons';
+import { Text } from '../typography';
+import { BreadcrumbLink, BreadcrumbLinkProps } from './BreadcrumbLink';
+
+interface BreadcrumbsProps {
+  className?: string;
+  items: BreadcrumbLinkProps[];
+}
+
+const BreadcrumbsBase = forwardRef<HTMLDivElement, BreadcrumbsProps>((props, ref) => {
+  const { items, ...rest } = props;
+
+  return (
+    <div ref={ref} {...rest}>
+      {items.map((item, index) => (
+        <React.Fragment key={`${index}-${item.to.toString()}`}>
+          {index > 0 && <IconSlash color="echoes-color-icon-subdued" />}
+
+          {index === items.length - 1 ? (
+            <ChildBreadcrumb className={item.className}>{item.linkElement}</ChildBreadcrumb>
+          ) : (
+            <BreadcrumbLink {...item} />
+          )}
+        </React.Fragment>
+      ))}
+    </div>
+  );
+});
+
+BreadcrumbsBase.displayName = 'BreadcrumbsBase';
+
+const ChildBreadcrumb = styled(Text)`
+  font: var(--echoes-typography-text-small-regular);
+`;
```

Comment:
You can replace the ChildBreadcrumb component with a simple span. It will inherit the font from the Breadcrumbs